docs/usage/remote-plugins.rst

```diff
@@ -45,3 +46,46 @@ but it can make asynchronous requests, i.e. passing ``async=True``.
 
 You need to run ``:UpdateRemotePlugins`` in Neovim for changes in the specifications to have effect.
 For details see ``:help remote-plugin`` in Neovim.
+
+If you are working on your plugin locally, it may be desirable to run the
+plugin from a git repository directly without the need of installing or
+symlinking it.
+For local plugin development, it's a good idea to use an isolated vimrc:
+
+.. code-block:: console
+
+    $ cat vimrc
```

Comment:
The `$` makes copy-pasting commands really annoying. Even if it's ambiguous, it's generally not a problem.